vision/realsense/camera_driver.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
How about we convert this to a readme, ie vision/realsense/README.md.
This is a markdown file where you can have code snippets, headers, ...
https://guides.github.com/features/mastering-markdown/